adoublef/noughts-and-crosses
# Vercel frontend deployment

- fix/merge conflict b (#25)
- update readme
- github test action
- update test
- remove test-cache
- mod tidy
- testing on ubuntu and 1.20.x only
- update deps
- remove download
- on pr only
- setup-v3
- add go mod tidy
- update deps
- exclude metadata
- second attempt
- add wait strategy
- mod tidy
- added dockerfile
- launched backend to fly
- added health endpoint on root
- rename env names
- update fly config with args
- added buildvcs flag
- added app to process
- revert toml config
- update to toml file
- update to parse redis url
- readme stuff
- updated port
- add host to port
- added pg
- fix linking
- add port
- fixed test
- added secrets for staging
- use second app instead of pr reviews
- delete conf pacakge
- mod tidy
- user_login_test works
- clean up server
- using chi's jwtauth module
- get started on auth
- auth http
- verify tests work
- update deps
- pass raw jwtauth type to two services
- fix bugs
- remove jwt package inplace for token package
- tests all pass
- services use new token.Client
- update main
- using new staging name
- update readme
- start publishsing to vercel
